src/Codeception/Template/Acceptance.php

```diff
@@ -99,6 +96,8 @@ public function setup()
         $this->gitIgnore($supportDir . DIRECTORY_SEPARATOR . '_generated');
         $this->sayInfo("Created test directories inside at $dir");
 
+        $this->addModulesToComposer(['WebDriver']);
```

Comment:
A special case for PHAR file is necessary.
When the phar file is used, composer.json file is not necessary and WebDriver and other modules are included to phar file.
Maybe something like this:
```php
if (!class_exists(\Codeception\Module\WebDriver:class)) {
  $this->addModulesToComposer(['WebDriver']);
}
```

Also when any modules is installed using composer, it effectively turns phar installation to composer installation.

Reply:
Done